src/serializer/serializer.js

```diff
@@ -63,10 +64,19 @@ export class Serializer {
       let forbiddenNames = realmPreludeGenerator.nameGenerator.forbiddenNames;
       traverseFast(ast, node => {
         if (!t.isIdentifier(node)) return false;
-
         forbiddenNames.add(((node: any): BabelNodeIdentifier).name);
         return true;
       });
+      traverseFast(ast, node => {
```

Comment:
By doing a walk over the AST in this way, you are counting what's statically present in the code. The intention of the referenced issue was to count which actually happens, and how often, while running the code in our the interpreter. Look at src/environment.js, there's a `evaluateAbstract` function which gets called for each AST node.

Reply:
Ok I will relook at this and try again.